dev-assist/src/main/resources/META-INF/plugin.xml

```diff
@@ -22,6 +22,7 @@
         <codeInsight.lineMarkerProvider language="JAVA"
                                         implementationClass="de.fraunhofer.iem.devassist.ui.markers.ErrorLineMarker"/>
         <toolWindow id="Dev-Assist" anchor="right" factoryClass="de.fraunhofer.iem.devassist.ui.SummaryToolWindow" icon="PluginIcons.DEV_ASSIST"/>
+        <notificationGroup id="Plugin-1" displayType="TOOL_WINDOW"/>
```

Comment:
Can you use a more descriptive name for the notification group? Plugin-1 doesn't give that much info.